### ISSUE TITLE
Bug - Disabled input field is visually similar to default

### ISSUE THREAD

**Issue by TrebBrennan** [AusDTO/gov-au-ui-kit#166]
## What I expected to see

A more dramatic visually identifiable difference between `input` and `input[disabled]`
### Additional information

Consider `border-style: dashed;`

**Comment by klepas** [comment]
This should be fixed as of #238; closing.
